### ISSUE TITLE
Cannot use Cmake after I use pact to install it

### ISSUE THREAD

**Issue by jingkangy91** [babun/babun#657]
I can use pact to install cmake successfully, like "pact install cmake", but after that, cmake is not working.
But in another PC, it can be used.

**Comment by cschram** [comment]
Does it just do nothing if you run it? Because that's the behavior I'm seeing, and the same happens for other programs, like gdb.

**Comment by jingkangy91** [comment]
Yes, doing nothing after run it and display nothing, and after add -V it will not show the version information. I think it's the version problem, cause I find some solution for problem that the gcc-core's version is not matched with g++'s. Maybe the version of cmake in pact is not matching with gcc.

**Comment by mcourteaux** [comment]
Yes I'm having the same problem. It exits with code 127, whatever I try.

**Comment by thyu** [comment]
I tried to run "cmake.exe" directly from the file explorer and it said "The procedure entry point acl_add_perm could not be located in the dynamic link library %BABUN_HOME\bin\cygarchive-13.dll."

I bet the either cmake is too new or libarchive is too old?

**Comment by alwyn** [comment]
Mine complains about cygwin1.dll
